### ISSUE TITLE
Add some debian autopkgtests

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#449]
See https://people.debian.org/~mpitt/autopkgtest/README.package-tests.html for background. autopkgtests appear to allow for testing of built packages as they are installed. I imagine the first two we could add would be a `notcurses-tester` and a `notcurses-demo` run using the installed binaries+data. This appears to carry some weight, see for instance:

```
2. We have introduced a new milestone between the Soft Freeze and the
Full Freeze, dubbed the Hard Freeze. During the Hard Freeze, key
packages and packages without (non-trivial) autopkgtests are
automatically blocked from migrating. These packages will require a
manual unblock from the release team to migrate, just like in the Full
Freeze.
```

so not a high priority by any means, but this would be nice to have.

**Comment by dankamongmen** [comment]
I've added one for both of `notcurses-tester` and `notcurses-demo`, and it will be present as of notcurses-1.6.1+dfsg1.1:

```
[schwarzgerat](0) $ cat debian/tests/control
Tests: notcurses-tester
Depends: notcurses-bin
Restrictions: allow-stderr

Tests: notcurses-demo
Depends: notcurses-bin
Restrictions: allow-stderr
[schwarzgerat](0) $
```
